tests/Integration/Behaviour/Features/Context/Domain/CmsPageFeatureContext.php

```diff
@@ -338,14 +320,14 @@ private function createCmsPageUsingCommand($cmsPageReference, array $data)
     {
         $command = new AddCmsPageCommand(
             (int) $data['id_cms_category'],
-            [$this->defaultLangId => $data['meta_title']],
-            [$this->defaultLangId => $data['head_seo_title']],
-            [$this->defaultLangId => $data['meta_description']],
-            [$this->defaultLangId => $data['link_rewrite']],
-            [$this->defaultLangId => $data['content']],
+            [$this->getDefaultLangId() => $data['meta_title']],
+            [$this->getDefaultLangId() => $data['head_seo_title']],
+            [$this->getDefaultLangId() => $data['meta_description']],
+            [$this->getDefaultLangId() => $data['link_rewrite']],
+            [$this->getDefaultLangId() => $data['content']],
             PrimitiveUtils::castStringBooleanIntoBoolean($data['indexation']),
             PrimitiveUtils::castStringBooleanIntoBoolean($data['active']),
-            [$this->defaultShopId]
+            [$this->getDe]
```

Comment:
@jolelievre typo

Reply:
Woups sorry, I didn't have time to re-read my own PR Thanks